packages/client/src/modules/i18n/index.web.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import i18n from 'i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
-import { reactI18nextModule, I18nextProvider } from 'react-i18next';
+import { I18nextProvider, reactI18nextModule } from 'react-i18next';
 
 import Feature from '../connector';
-import { MenuItem, LanguagePicker } from '../../modules/common/components/web';
+import { LanguagePicker, MenuItem } from '../../modules/common/components/web';
```

Comment:
What is the point in changes like these?

Reply:
I ordered dependencies in alphabetical order.